openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -31,6 +38,116 @@ def get_identity(self, data):
         except AttributeError:
             return None
 
+class AccountFullUserProfileReadOnlySerializer(serializers.Serializer):
+    """
+    Class that serializes the portion of User model needed for account information.
+    """
+    def __init__(self, *args, **kwargs):
+        # Don't pass the 'configuration' arg up to the superclass
+        self.configuration = kwargs.pop('configuration', None)
+        if not self.configuration:
+            self.configuration = settings.ACCOUNT_VISIBILITY_CONFIGURATION
+
+        # Don't pass the 'admin_fields' arg up to the superclass
+        self.admin_fields = kwargs.pop('admin_fields', False)
+
+        # Instantiate the superclass normally
```

Comment:
This comment seems superfluous. I wouldn't want to have such a comment every time we call ``super``.